.semaphore/semaphore.yml

```diff
@@ -487,6 +487,127 @@ blocks:
       commands:
       - make ci
 
+- name: "cni-plugin: Build Windows binaries"
+  run:
+    when: "branch =~ '.*' or change_in(['/*', '/cni-plugin/', '/libcalico-go/'], {exclude: ['/**/.gitignore', '/**/README.md', '/**/LICENSE']})"
+  dependencies: []
+  task:
+    prologue:
+      commands:
+      - cd cni-plugin
+    jobs:
+    - name: "build Windows binaries"
+      commands:
+      - make build && make bin/windows/win-fv.exe
```

Comment:
do we need make build here? What is that for? That will make the linux binaries as well, which we shouldn't do for the windows job.